nautobot/docs/user-guide/administration/tools/nautobot-server.md

```diff
@@ -686,21 +686,26 @@ Done.
 
 ### `runjob`
 
-`nautobot-server runjob [job]`
+`nautobot-server runjob --username <username> [--local] [--data <data>] <job>`
 
-Run a job (script, report) to validate or update data in Nautobot.
-
-`--commit`  
-Commit changes to DB (defaults to dry-run if unset). `--username` is mandatory if using this argument.
+Run a job (script, report) to validate or update data in Nautobot. The Job name must be in the Python module form: `<module_name>.<JobClassName>`.
```

Comment:
```suggestion
Run a job (script, report) to validate or update data in Nautobot. The Job name must be in the Python module form: `<module_name>.<JobClassName>`. You can find this under the Jobs detail view on the "Class Path" row.
--


<br class="Apple-interchange-newline">
```

Reply:
@itdependsnetworks was this part intentional? What does this add?

```
<br class="Apple-interchange-newline">
```

Reply:
It was not... by bad, I have updated the comment.